File: Nivel Basico/softwareLibre.js
// https://omegaup.com/arena/problem/RedistribuyendoSoftwareLibre/
process.stdin.on("data", (data) => {
   let entrada = data.toString().split("\n");
   solve(entrada);
   process.exit();
});

function solve(entrada){
   // console.log(entrada);
   for(let i = 1; i < entrada.length; ++i){
      let numero = parseInt(entrada[i]);
      if(numero >= 3) process.stdout.write("Ok\n");
      else process.stdout.write("No\n");
   }
}

File: Nivel Basico/softwarelibre.cpp
// https://omegaup.com/arena/problem/RedistribuyendoSoftwareLibre/
#include <bits/stdc++.h>

using namespace std;

int main(){
   cin.tie(nullptr);
   ios_base::sync_with_stdio(false);
   int n, number; 
   scanf("%d", &n);
   while (n--){
      scanf("%d", &number);
      if(number >= 3) printf("Ok\n");
      else printf("No\n");
   }
   return 0;
}